/*
 * This file was automatically generated by EvoSuite
 * Sun Apr 08 22:44:58 GMT 2018
 */

package java_programs;

import org.junit.Test;
import static org.junit.Assert.*;
import java_programs.POSSIBLE_CHANGE;

public class POSSIBLE_CHANGE_ESTest {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      POSSIBLE_CHANGE pOSSIBLE_CHANGE0 = new POSSIBLE_CHANGE();
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      int int0 = POSSIBLE_CHANGE.possible_change((int[]) null, (-2171));
      assertEquals(0, int0);
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      // Undeclared exception!
      try { 
        POSSIBLE_CHANGE.possible_change((int[]) null, 1);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
      }
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      int[] intArray0 = new int[3];
      intArray0[0] = 3514;
      intArray0[1] = 2183;
      intArray0[2] = 1;
      int int0 = POSSIBLE_CHANGE.possible_change(intArray0, 1);
      assertEquals(1, int0);
  }
}
